chroma_core/models/client_mount.py

```diff
@@ -18,7 +18,9 @@
 
 class LustreClientMount(DeletableStatefulObject):
     host = models.ForeignKey("ManagedHost", help_text="Mount host", related_name="client_mounts", on_delete=CASCADE)
-    filesystem = models.ForeignKey("ManagedFilesystem", help_text="Mounted filesystem", on_delete=CASCADE)
+    filesystem = models.CharField(
+        max_length=CHARFIELD_MAX_LENGTH, help_text="Mounted filesystem", null=False, blank=False
+    )
```

Comment:
This should have the same restrictions as the `name` field on `ManagedFilesystem`:

https://github.com/whamcloud/integrated-manager-for-lustre/blob/65490babfbae9d378d52be74f637923ef4850e47/chroma_core/models/filesystem.py#L44-L48